src/PrestaShopBundle/Install/Install.php

```diff
@@ -346,9 +346,16 @@ public function updateSchema()
         $schemaUpgrade = new UpgradeDatabase();
         $schemaUpgrade->addDoctrineSchemaUpdate();
         $output = $schemaUpgrade->execute();
+        $schemaUpdateOutput = $output['prestashop:schema:update-without-foreign']['output'];
+        if (_PS_MODE_DEV_ && substr($schemaUpdateOutput, 2, 9) === '[WARNING]') {
```

Comment:
```suggestion
        if ($this->isDebug && substr($schemaUpdateOutput, 2, 9) === '[WARNING]') {
```

Reply:
I had not seen the variable :) thank you